### PR TITLE
Release notes 2.16.3

### PR DESCRIPTION
**Pull request by serinamarie**
[Preview](https://github.com/PrefectHQ/prefect/blob/release-notes-2.16.3/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
...
+- https://github.com/PrefectHQ/prefect/pull/12187
+- https://github.com/PrefectHQ/prefect/pull/12182
+- https://github.com/PrefectHQ/prefect/pull/12219
+
+### Enhancements
+- Adds `on_running` state change hook — https://github.com/PrefectHQ/prefect/pull/12153
+- Add flow run state data to flow run graph API — https://github.com/PrefectHQ/prefect/pull/12130 
+- Add schedules shorthand support to `Deployment.build_from_flow` — https://github.com/PrefectHQ/prefect/pull/12181
+- Send flow run and task run retry logs to API — https://github.com/PrefectHQ/prefect/pull/12211
+- Allow creating autonomous task runs via `__call__` — https://github.com/PrefectHQ/prefect/pull/12158
```
Comment (zzstoatzz) [inline RELEASE-NOTES.md:26]: this should probably go in experimental as well, sorry, I should have labeled it accordingly

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+- Add the flow run graph states layer and handle selection — https://github.com/PrefectHQ/prefect/pull/12166 - asked Brandon
+- Add a paused deployment filter and update `is_schedule_active` filter — https://github.com/PrefectHQ/prefect/pull/12202
+- Updating client-side schemas for automation triggers to reflect updates in Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/12205
+- Update aggregate worker metadata view — https://github.com/PrefectHQ/prefect/pull/12183
```
Comment (zzstoatzz) [inline RELEASE-NOTES.md:30]: this should have been labeled with `maintenance` - I will be better 🙃 
```suggestion
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+- Address two memory leak in concurrency management — https://github.com/PrefectHQ/prefect/pull/12141
+
+### Experimental
+- Add Job Variables tab to the Flow Runs page — https://github.com/PrefectHQ/prefect/pull/12206
+- Add support for calling `.map` for autonomous task runs — https://github.com/PrefectHQ/prefect/pull/12171
+- Simplify already authenticated logic for `prefect cloud login` — https://github.com/PrefectHQ/prefect/pull/12209
+- Add better parity with Prefect Cloud for task scheduling protocol - https://github.com/PrefectHQ/prefect/pull/12212
+
+### Integrations
+- Add `prefect-soda-cloud` integration — https://github.com/PrefectHQ/prefect/pull/12208
```
Comment (zzstoatzz) [inline RELEASE-NOTES.md:41]: since this is just adding to the collections catalog maybe this should be a docs one?

Comment (serinamarie) [reply RELEASE-NOTES.md:41]: hm, sure i'm indifferent!

Comment (zzstoatzz) [reply RELEASE-NOTES.md:41]: yeah same, its prob fine as is!

Comment (serinamarie) [reply RELEASE-NOTES.md]: applied in [868d8f0](https://github.com/PrefectHQ/prefect/pull/12217/commits/868d8f0ed66daa011f86701bf16e509fef0b5a30)

**Review by zzstoatzz — APPROVED** [review]
👾

**Pull request merged**